examples/all_conditions.rs

```diff
@@ -2,6 +2,7 @@
 //! Press keys from the number row on the keyboard to trigger actions and observe the output in console.
 
 use bevy::{ecs::spawn::SpawnWith, log::LogPlugin, prelude::*};
+use bevy_enhanced_input::condition::cooldown::Cooldown;
```

Comment:
Ah, forgot to ask: add the type to `prelude`, so you won't need to import it here and in `src/lib.rs`.